src/applications/personalization/profiles-wrapper/ProfilesWrapper.jsx

```diff
@@ -2,31 +2,21 @@ import React from 'react';
 import { connect } from 'react-redux';
 import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
 import Profile2Router from 'applications/personalization/profile-2/components/Profile2Router';
-import ProfileOneWrapper from 'applications/personalization/profile360/containers/VAProfileApp';
```

Comment:
I guess in another PR we could remove this entire `ProfileWrapper` component since its primary role was to decide if we show Profile 1 or Profile 2.